src/plugins/intel_cpu/src/transformations/snippets/x64/pass/brgemm_to_brgemm_cpu.hpp

```diff
@@ -15,11 +15,11 @@ namespace pass {
  * @brief The pass decompose Snippets Brgemm to specific subgraph that depends on ISA and input precisions:
  *        - f32|f32 without transpose_b:
  *                   BrgemmCPU
- *        - u8|i8 or bf16|bf16 (non-AMX system):
+ *        - u8|i8 or bf16|bf16 (non-AMX system) or i8|i8 (with avx2_vnni_2 support):
```

Comment:
should we add f32 with transpose_b as well?

Reply:
Added